### DISCUSSION TITLE
OutOfMemory but I have change memory_limit to -1 & 2048MB ..

### DISCUSSION THREAD

**Discussion by subskill-tsoulier** [sulu/sulu#6947] [Q&A] [answered]
Hi, 

In a first time, I'm french and I'm sorry if my english is bad.

Device :
MacMini i5 8Go 2014.

PhP : 
7.4.2

( I go tried with PhP 8.1.12)

Command Installation :
`composer create-project sulu/skeleton sulu`
&
`composer create-project sulu/skeleton sulu -n`

Actual error : 
![Capture d’écran 2022-12-15 à 11 39 19](https://user-images.githubusercontent.com/114140953/207838292-e148bd94-ad3d-44b4-9c53-6518c42c6432.png)

I have setup my memory_limit at 512MB, 1024MB, 2048MB & memory_limit= -1 and I have this error.
I have restart my apache & MySQL service after this changement and check with php -i

Thank for you're futur help ! <3

**Comment by alexander-schranz** [comment]
@subskill-tsoulier that sounds really strange. You definitly should try out PHP 8.1 or PHP 8.2 as PHP 7.4 is not longer supported. If you still run into this problem share your `php -i` and `php -m` output.

I know @alengodev had some strange php extension on his hoster last time running into strange problems. Not remembering the name of it 🤔

**Reply by alexander-schranz** [reply, marked as answer]
Also try to use:

```php
php -d memory_limit=-1 bin/adminconsole cache:clear
php -d memory_limit=-1 bin/websiteconsole cache:clear
```

to build up the caches. As it seems `134217728 bytes` is still the `128 MB` and for creating the `dev` container you require more. Maybe something wrong with your `PHP` `CLI` setting and you only changed the webserver memory_limit.

**Reply by subskill-tsoulier** [reply]
Hi, sorry my longer response.
I have reinstall PhP to 8.2.0 and now it's working, I think that PhP Version or configuration, btw I have change my memory_limit too.
Thank for you're help !

**Comment by alengodev** [comment]
We had problems with the ioncube_loader php extension in php 8.1 - deactivated it and all works fine.
